### PR TITLE
add proguard rules

### PR DESCRIPTION
**Pull request by ebsangam**
Solves issue on Android where user needs to add proguard rules. This PR will add proguard rules to the plugin.

### CODE REVIEW

**Review by IVLIVS-III — COMMENTED** [review]
Did I understand correctly, that after merging this PR users of the `device_calendar` plugin no longer need to add that line to their proguard file anymore?

What happens if a user uses other rules in their proguard file, will they be merged by proguard?

If so, please also change the Readme, that it no longer says users must add that line to their proguard files.

File: example/android/build.gradle [hunk 1]
```diff
@@ -24,6 +24,6 @@ subprojects {
     project.evaluationDependsOn(':app')
 }
 
-task clean(type: Delete) {
+tasks.register("clean", Delete) {
```
[the remark this answers is not in the document]
Comment (ebsangam) [reply example/android/build.gradle:27]: This was automatically updated from Groovy to Kotlin DSL  when running example.

**Comment by ebsangam** [comment]
> Did I understand correctly, that after merging this PR users of the `device_calendar` plugin no longer need to add that line to their proguard file anymore?
> 
> What happens if a user uses other rules in their proguard file, will they be merged by proguard?
> 
> If so, please also change the Readme, that it no longer says users must add that line to their proguard files.

1. App will no longer needs to add proguard rules.
2. They will be merged.
3. Updating ...

File: example/android/build.gradle (hunk 1 above)
[the remark this answers is not in the document]
Comment (IVLIVS-III) [reply example/android/build.gradle:27]: Okey, great thanks for the clarification.

**Review by IVLIVS-III — APPROVED** [review]
LGTM

**Pull request merged**